server/eclipse-project/src/main/java/us/freeandfair/corla/report/StateReport.java

```diff
@@ -630,14 +628,14 @@ public Workbook generateExcelWorkbook() {
         if (discrepancies.isEmpty()) {
           cell.setCellValue("No Discrepancies Recorded");
         } else {
-          cell.setCellValue("Discrepancies Recorded by Audit Reason");
-          for (final AuditReason r : listed_reasons) {
+          cell.setCellValue("Discrepancies Recorded");
+          for (final AuditSelection s : listed_selections) {
             cell = row.createCell(cell_number++);
             cell.setCellType(CellType.NUMERIC);
             cell.setCellStyle(integer_style);
             final int cell_value;
-            if (discrepancies.containsKey(r)) {
-              cell_value = discrepancies.get(r);
+            if (discrepancies.containsKey(s)) {
```

Comment:
`getOrDefault()` might be useful throughout here to get rid of the `if`/`then` switches, if you like.

Reply:
Indeed, I’ll take a look at that.

Reply:
Actually, it doesn't make sense here because here, we don't print the columns if they don't exist (rather than just printing them as 0). That's a choice that could be revisited, of course.